origami/exceptions.py

```diff
@@ -10,14 +10,14 @@ def __init__(self, message=''):
 
 class MismatchTypeException(OrigamiException):
     """
-    These exceptions are caused during mismatched types.
+    These exceptions are cause during mismatched types.
```

Comment:
There was some mistake during the rebase, this can you merge this, for now, I will fix it in the next one. This will be too much work for now :sweat_smile: